src/redux/modules/auth.js

```diff
@@ -12,7 +12,7 @@ const UPDATE_TOKEN   = 'Auth/UPDATE_TOKEN';
 export const types = { LOGIN, LOGIN_SUCCESS, LOGOUT_SUCCESS, LOGOUT };
 
 /* Reducer */
-const initialState = { user: null, token: null, error: null, isKcReady: false };
```

Comment:
why did you remove user from here?

You want to check if it is undefined? vs null?

Reply:
yes I need case that user is undefined